admin/installation.rst

```diff
@@ -241,11 +241,11 @@ However, we test and officially support only some of them.
 Supported platforms for |product_name| server and agent:
 
    * Debian 10 (Buster), 11 (Bullseye), 12 (Bookworm)
-   * Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal Fossa), 22.04 LTS (Jammy Jellyfish)
+   * Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal Fossa), 22.04 LTS (Jammy Jellyfish), 24.04 ( Noble )
```

Comment:
Let's make it like this for consistency:

Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal), 22.04 LTS (Jammy), 24.04 (Noble)